assets/js/modules/analytics/datastore/setup-flow.js

```diff
@@ -40,57 +40,31 @@ const baseSelectors = {
 			return SETUP_FLOW_MODE_LEGACY;
 		}
 
-		// Check to see if the Admin API is working—if it's `undefined` the request
-		// is loading, but if it's `false` we should also use the legacy analytics
-		// because the API isn't working properly.
+		// Check to see if the Admin API is working—if it's `false` we should
+		// use the legacy analytics because the API isn't working properly.
 		const isAdminAPIWorking = select( MODULES_ANALYTICS_4 ).isAdminAPIWorking();
-
-		if ( isAdminAPIWorking === undefined ) {
-			return undefined;
-		}
```

Comment:
I think this (and the `isAdminAPIWorking` ability to differentiate between `undefined` and `true`) shouldn't be removed, it needs to be maintained. Otherwise, the selector returns `true` even though it doesn't even know (yet) that the GA4 Admin API _actually_ works.